### DISCUSSION TITLE
Linking equations in MoBi

### DISCUSSION THREAD

**Discussion by Sarang-Mishra** [Open-Systems-Pharmacology/Forum#1199] [Questions & Problems]
Dear all,
I am working on a drug  metabolized a non-CYP enzyme. I want to know how can an eqaution can be linked with other eqation/reaction in MoBi. Additionally, if anyone know about any example of equation for time- dependent, concentration-dependent inhibtion of enzyme.

Thanks,
Sarang

**Comment by StephanSchaller** [comment]
@Sarang-Mishra , from what you describe, this should be doable in PK-Sim with no need to swap to MoBi.
